packages/react-components/lib/tasks/task-table-datagrid.tsx

```diff
@@ -134,11 +135,24 @@ export function TaskDataGridTable({
     {
       field: 'requester',
       headerName: 'Requester',
-      width: 150,
+      minWidth: 160,
       editable: false,
-      valueGetter: (params: GridValueGetterParams) =>
-        params.row.booking.requester ? params.row.booking.requester : 'unknown',
-      flex: 1,
+      renderCell: (cellValues) => {
+        return (
+          <Stack direction="row" alignItems="center" gap={1}>
+            {cellValues.row.booking.requester &&
+            cellValues.row.booking.requester.includes('scheduled') ? (
+              <ScheduleIcon />
+            ) : (
+              <UserIcon />
+            )}
+            <Typography variant="body1">
+              {cellValues.row.booking.requester ? cellValues.row.booking.requester : 'unknown'}
```

Comment:
We can probably just display `stub` instead of `stub__scheduled` here, the different icon should be a good enough indicator

Reply:
Sure!
Done 17b513d8de25bf91f8fe4136401a830095bdf5fe